mopidy_local/storage.py

```diff
@@ -199,15 +193,16 @@ def _extract_images(self, uri, tags):
             except Exception as e:
                 logger.warning("Error extracting images for %r: %r", uri, e)
         # look for external album art
-        path = translator.local_uri_to_path(uri, self._media_dir)
-        # replace brackets with character classes for use with glob
-        dirname = os.path.dirname(path).replace("[", "[[]")
```

Comment:
What was this doing?

Reply:
Subset of what https://docs.python.org/3/library/glob.html#glob.escape does?

Reply:
Ah thanks OK I see now why we don't need it any more.